ui/app/manager/src/pages/page-map.ts

```diff
@@ -252,6 +281,10 @@ export class PageMap extends Page<MapStateKeyed> {
 
                 this._store.dispatch(setAssets(assets));
 
+                if (this._map) {
+                    this.updateMarkers();
+                }
```

Comment:
Shouldn't this ` if(this._map) { ` condition be inside the `updateMarkers()` function?

(my single responsibility code guidelines triggered this 😂)